Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3
+
+ARG arg_HOME=/opt/sonos-huter
+ENV HOME=$arg_HOME
+
+RUN mkdir ${HOME}
```

Comment:
Should this be `mkdir -p` ?

Reply:
Yes.

Reply:
@alexander-lloyd  Is this satisfactory, or have I introduced an SQL injection?